SERVER/CONTROLLERS/userAcc_Controler.js

```diff
@@ -139,12 +140,17 @@ class userAccountControler{
     }
 
     viewAllMentors(req, res){
-        const accs= accounts.AllAccounts;
-        const mentor_users=accs.filter(user=>user.isAmentor===true);
-            return res.status(200).json({
-                status: 200,
-                data: mentor_users
+        const Is_a_mentor=true;
+        pool.connect((err, client, done) => {
```

Comment:
Move any call to the database into service files. Let your controllers only receive requests call other methods and then return a response with the results